app/models/contact.rb

```diff
@@ -14,6 +14,9 @@ class Contact < ActiveRecord::Base
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :permalink, uniqueness: {
+    # FIXME: I believe your validation message here is going to read:
```

Comment:
The error message is misleading to an end-user who may not understand how a permalink is derived from first & last name. While I agree that this error message should be improved, I'd rather not use a gem for it. Hmm, maybe a custom validation method, making use of `errors[:base]`, would work better?